systemd/manager.go

```diff
@@ -106,6 +106,14 @@ func (m *systemdUnitManager) Load(name string, u unit.UnitFile) error {
 	if err != nil {
 		return err
 	}
+	if len(u.Contents["Install"]) > 0 {
```

Comment:
Well, my concern is that this condition seems misleading at best: it suggests that we are skipping empty sections on purpose. (While only implicitly skipping entirely missing sections.) I don't think that's really an intentional choice?

From your other comments, I understand that the way fleet parses the units currently, empty sections will never show up here; so "is present" and "has a length" are effectively the same for now. However, that is an implementation detail that might change at some point -- and either way, it's always better to be explicit about intentions...